pkgs/misc/ghostscript/default.nix

```diff
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
   patches = [
     ./urw-font-files.patch
     ./doc-no-ref.diff
+    ./0001-For-ICC-profile-validation-have-cups-id-iteself-as-D.patch
```

Comment:
I didn't use `fetchpatch`, as the source url `http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=bc3df0773fcc` caused problems.

Reply:
Add the source url in the commit message and state that it must me included into nixpkgs due to invalid characters.